From 1599ecb3e81bdb9be4f54fc1e890b4d7ae3fa338 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Fri, 13 Sep 2019 16:15:01 -0300
Subject: [PATCH] [FIO toup] docker-apps: remove extra app argument from
 docker-app

Docker app can be made available as a docker cli plugin (depends on
docker experimental, not enabled by default) or as standalone.
Standalone build means that the extra 'app' argument is not required, so
remove from the default command line parsing.

This is not yet handling docker-app as a docker cli plugin.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 src/libaktualizr/package_manager/dockerappmanager.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libaktualizr/package_manager/dockerappmanager.cc b/src/libaktualizr/package_manager/dockerappmanager.cc
index b9d15a88..f1c599ae 100644
--- a/src/libaktualizr/package_manager/dockerappmanager.cc
+++ b/src/libaktualizr/package_manager/dockerappmanager.cc
@@ -13,7 +13,7 @@ struct DockerApp {
   bool render(const std::string &app_content, bool persist) {
     auto bin = boost::filesystem::canonical(app_bin).string();
     Utils::writeFile(app_root / (name + ".dockerapp"), app_content);
-    std::string cmd("cd " + app_root.string() + " && " + bin + " app render " + name);
+    std::string cmd("cd " + app_root.string() + " && " + bin + " render " + name);
     if (!app_params.empty()) {
       cmd += " -f " + app_params.string();
     }
-- 
2.23.0

